semcore/data-table/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 CHANGELOG.md standards are inspired by [keepachangelog.com](https://keepachangelog.com/en/1.0.0/).
 
+## [4.32.2] - 2024-03-27
+
+### Fixed
+
+- Prevent Table.Header to be focused.
```

Comment:
Let's just say that you made it unfocusable

Reply:
fixed